### PR TITLE
Update BUILD_LINUX.md

### PR DESCRIPTION
**Pull request by MarcusLlewellyn**
- Fix python installation instructions.
- Remove redundant instructions.
- Since we are not using tags, instruct users to checkout `kasen/core`.
- Add instructions for a faster compile.
- Add optional instruction for the `HIFI_VCPKG_BASE` environment variable.
- Style fixes.

### CODE REVIEW

File: BUILD_LINUX.md [hunk 1]
```diff
@@ -143,6 +148,12 @@ Running interface:
 
 Go to localhost in the running interface.
 
+#### Notes
+
+If your goal is to setup a development environment, it is desirable to set the
```
Comment (ctrlaltdavid) [inline BUILD_LINUX.md]: setup -> set up

File: BUILD_LINUX.md [hunk 2]
```diff
@@ -14,60 +14,58 @@ sudo add-apt-repository universe
...
 ```bash
-sudo apt-get -y install libpulse0 libnss3 libnspr4 libfontconfig1 libxcursor1 libxcomposite1 libxtst6 libxslt1.1
-```
-1.  Install Python 3 and required packages
-```bash
-sudo apt-get install python3 python3-distro
+sudo apt-get install python python3 python3-distro
 ```
-1.  Install node, required to build the jsdoc documentation
+-  Install node, required to build the jsdoc documentation
```
Comment (ctrlaltdavid) [inline BUILD_LINUX.md]: ":" missing from end.

File: BUILD_LINUX.md [hunk 3]
```diff
@@ -14,60 +14,58 @@ sudo add-apt-repository universe
...
 ```bash
 sudo apt-get -y install libpulse0 libnss3 libnspr4 libfontconfig1 libxcursor1 libxcomposite1 libxtst6 libxslt1.1
 ```
-1.  Misc dependencies
+-  Misc dependencies
 ```bash
 sudo apt-get install libasound2 libxmu-dev libxi-dev freeglut3-dev libasound2-dev libjack0 libjack-dev libxrandr-dev libudev-dev libssl-dev zlib1g-dev
 ```
-1.  To compile interface in a server you must install:
+-  Install Python 3 and required packages
```
Comment (ctrlaltdavid) [inline BUILD_LINUX.md]: ":" missing from end. Similarly for other instructions, above.

**Comment by ctrlaltdavid** [comment]
BUILD_LINUX_CHEATSHEET.md also needs updating (e.g., it references debian.highfidelity.com).

**Comment by MarcusLlewellyn** [comment]
> BUILD_LINUX_CHEATSHEET.md also needs updating (e.g., it references debian.highfidelity.com).

I'm wondering if we shouldn't just omit that document for now. We aren't hosting or distributed DEB packges right now.

File: BUILD_LINUX.md [hunk 4]
```diff
@@ -150,7 +150,7 @@ Go to localhost in the running interface.
 
 #### Notes
 
-If your goal is to setup a development environment, it is desirable to set the
+If your goal is to setup a development environment, it is desirable to set up the
```
Comment (ctrlaltdavid) [inline BUILD_LINUX.md]: setup -> set up

Comment (MarcusLlewellyn) [reply BUILD_LINUX.md]: Fixed.

File: BUILD_LINUX.md (hunk 3 above)
Comment (MarcusLlewellyn) [reply BUILD_LINUX.md]: Fixed

File: BUILD_LINUX.md (hunk 2 above)
Comment (MarcusLlewellyn) [reply BUILD_LINUX.md]: Fixed

File: BUILD_LINUX.md (hunk 4 above)
Comment (MarcusLlewellyn) [reply BUILD_LINUX.md]: Actually I'm confused. :)

Comment (ctrlaltdavid) [reply BUILD_LINUX.md]:
```
If your goal is to setup a development environment, …
``
should be
```
If your goal is to set up a development environment,
```

The extra "up" you subsequently added should be removed.

Comment (MarcusLlewellyn) [reply BUILD_LINUX.md]: Okay, fixed.

**Pull request merged**